### DISCUSSION TITLE
Isn't the "toggle navigation" arrow in the book theme confusing?

### DISCUSSION THREAD

**Discussion by fmaussion** [executablebooks/meta#546] [Ideas] [answered]
I always want to use the arrow on the top as a "go back" (or "navigate back to previous page") button. I almost never want to toggle the menu away. Is it just me or do others here agree?

![image](https://user-images.githubusercontent.com/10050469/141684894-55cb4e4c-2175-4ee5-b81a-f5cfffa3b529.png)

**Comment by choldgraf** [comment, marked as answer]
I think you're right - there is some conversation about this here: https://github.com/executablebooks/sphinx-book-theme/issues/355

Another easy thing we could go is just use a single icon that is a hamburger stack or something.

**Reply by fmaussion** [reply]
Sorry I missed this issue! Totally agree with your suggested changes there.
